src-backbone/app/js/models/session.js

```diff
@@ -96,6 +97,46 @@ let SessionModel = Backbone.Model.extend({
         });
     },
 
+    updateInformation: function(formData) {
+        let self = this;
+        let json = {};
+        formData.forEach(function(item) {
+            if (item.value !== "") {
```

Comment:
possibly silly question: will this allow `null`? I think `if (item.value)` will be truthy for non-`null`, non-empty strings.